webui/react/src/services/api.ts

```diff
@@ -157,15 +173,17 @@ export const openOrCreateTensorboard = async (
   return createTensorboard(params);
 };
 
-export const killTask = async (task: AnyTask, cancelToken?: CancelToken): Promise<void> => {
-  if (isExperimentTask(task)) {
-    return await killExperiment({ cancelToken, experimentId: parseInt(task.id) });
+export const killCommandTask = async (task: CommandTask): Promise<void> => {
```

Comment:
**nit:** Let's keep this as `killTask`. Historically task was called Command, but because there is an actual Command type within Commands, Task was a better name to distinguish it.

Reply:
Done 👍